cluster/examples/app/cockroach/main.go

```diff
@@ -0,0 +1,62 @@
+package main
+
+import (
+	"database/sql"
+	"fmt"
+	"log"
+
+	_ "github.com/lib/pq"
+)
+
+const (
+	host     = "127.0.0.1"
+	port     = 5433
+	user     = "root"
+	password = "root"
+	dbname   = "bank"
+)
+
+func main() {
```

Comment:
It's a bit unexpected to have one of the examples be a golang app that needs to be compiled.  Is there any reuse from the typical loadgen app that rook cockroach uses? https://rook.io/docs/rook/v1.1/cockroachdb.html#example-app

if not, is there documentation for how to use these example apps? is there a maintenance burden you are concerned about?

Reply:
Yep I believe that could easily be reused :) I am going to update these examples with the Yugabyte one as well. Would you prefer that these examples either be removed or completed before merge of this PR?

Reply:
Are you saying that for v0.4 timeframe, you'll have yaml examples, instead of golang code examples that have to be compiled and deployed to run?

If I understand that correctly, I'd like to just do that in this PR if we can.  If not, then let's remove the golang code ones and then in a follow-up PR add the yaml ones.